doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -498,7 +498,6 @@ in a section.)
 
 When using ``-vos``, proofs in sections with :cmd:`Proof using` are skipped.  Proofs
 in sections without :cmd:`Proof using` are fully processed (much slower) and will raise a warning.
```

Comment:
We should still check though if this claim "will raise a warning" is true or false. If it is false, then we should remove it as well.

Reply:
No, no warning.  My test:

`vos.v` is
```
Section A.
Theorem foo : True.
Proof.
apply I.
Qed.
End A.
```

```
$ coqc -vos vos.v
jfehrle@LAPTOP-LB5T3R83:/home/proj/coq2$ ls vos.*
vos.v  vos.vos
```